pupil_src/shared_modules/reference_surface.py

```diff
@@ -84,7 +85,7 @@ def __init__(self, g_pool, name="unnamed", saved_definition=None):
         self.camera_pose_3d = None
         self.use_distortion = True
 
-        self.uid = str(time())
+        self.uid = "{:04}".format(random.randint(0,9999))
```

Comment:
Nice use of zero-padding!

Reply:
please replace this with a proper uuid or time.time with stripped '.'

Reply:
The idea behind this change was to make sure that generated file names, which contain the `uid`, are more manageable. My thinking was that a for digit random number should be more than enough in this case while being very manageable. Would you still prefer the longish but more unique timestamp?

Reply:
I think it would be worth it to check out the uuid python module and see if it contains anything helpful for this case.